app/src/main/java/org/simple/clinic/di/AppModule.kt

```diff
@@ -28,19 +28,27 @@ import org.simple.clinic.storage.StorageModule
 import org.simple.clinic.sync.SyncModule
 
 @Module(includes = [QrModule::class, SyncModule::class, NetworkModule::class, StorageModule::class, LoginModule::class, RegistrationModule::class])
-class AppModule(private val appContext: Application, private val databaseName: String = "red-db") {
+open class AppModule(private val appContext: Application, private val databaseName: String = "red-db") {
 
   @Provides
   fun appContext(): Application {
     return appContext
   }
 
+  @Provides
```

Comment:
`@Provides` is not necessary since we aren't injecting it anywhere, right? I think this can be a `protected val` parameter provided in the constructor.